architecture/trusted_applications.rst

```diff
@@ -81,6 +81,117 @@ filesystem, and ``tee-supplicant`` will take care of passing them to be checked
 and loaded by the Secure World OP-TEE core. Note that this type of TA isn't
 encrypted.
 
+REE filesystem TAs come in two formats, the legacy TA and the bootstrap TA.
```

Comment:
Maybe say that `scripts/sign.py` uses the boostrap TA format since version 3.7.0?

Reply:
I'll fix.